keel-api-jackson/keel-api-jackson.gradle.kts

```diff
@@ -26,6 +26,9 @@ dependencies {
   implementation("org.springframework:spring-context")
   implementation("org.springframework.boot:spring-boot-autoconfigure")
 
+  // this is really just so we can access `configuredObjectMapper`
+  testImplementation(project(":keel-core"))
```

Comment:
I guess the question I have is whether it's worth moving _all_ the Jackson configuration to this new module, or whether there's value in having it specifically dedicated to supporting the API types. 

Any thoughts on that?

Reply:
I have no idea.